packages/serverpod/lib/src/authentication/authentication_info.dart

```diff
@@ -11,8 +11,16 @@ typedef AuthenticationHandler = Future<AuthenticationInfo?> Function(
 /// Holds the id for an authenticated user and which [scopes] it can access.
 /// Allowed scopes are defined for each [Endpoint].
 class AuthenticationInfo {
-  /// Id for an authenticated user.
-  final int userId;
+  /// Integer Id for an authenticated user.
+  ///
+  /// Will throw in case the user ID of the authenticated user is not an integer.
+  @Deprecated(
+    'Use the authentication packages typed getter on `Session` instead (from `serverpod_auth` or `serverpod_auth_user`)',
```

Comment:
Clarify that `session.userId` should now be available to them (in auto-complete, might require `serverpod_auth` import for now (or `serverpod_auth_user` in the future)).